src/main/java/trivia/App.java

```diff
@@ -230,64 +249,66 @@ else if (j == 1) {
         );
 
-
+        // Funcion que basada en la pregunta que se quiere editar, la busca con sus respectivas opciones
         post ("/adminQuestions", (request, response) -> {
 
-            openDB();
-            List<Question> cambiar = Question.where("id = "+ Integer.parseInt(request.queryParams("id_question")));
+         openDB();
+         List<Question> cambiar = Question.where("id = "+ Integer.parseInt(request.queryParams("opciones")));
 
-            Question pregunta = cambiar.get(0);
+         Question pregunta = cambiar.get(0);
 
-            preguntas.put("ID", pregunta.getInteger("id"));
-            preguntas.put("pregunta", pregunta.get("pregunta"));
-            preguntas.put("opcion 1", pregunta.getString("respuestaCorrecta"));
-            preguntas.put("opcion 2", pregunta.getString("wrong1"));
-            preguntas.put("opcion 3", pregunta.getString("wrong2"));
-            preguntas.put("opcion 4", pregunta.getString("wrong3"));
-            preguntas.put("activada", pregunta.getString("active"));
+         preguntas.put("ID", pregunta.getInteger("id"));
+         preguntas.put("pregunta", pregunta.get("pregunta"));
+         preguntas.put("opcion 1", pregunta.getString("respuestaCorrecta"));
+         preguntas.put("opcion 2", pregunta.getString("wrong1"));
+         preguntas.put("opcion 3", pregunta.getString("wrong2"));
+         preguntas.put("opcion 4", pregunta.getString("wrong3"));
+         preguntas.put("activada", pregunta.getString("active"));
 
-            closeDB();
+         closeDB();
 
-            response.redirect("./changeQuestion");
+         response.redirect("./changeQuestion");
 
-            return null;
-        }); 
+         response.redirect("./adminMenu");
+         return null;
+       });
 
-        post ("/changeQuestion", (request, response) -> {
+       // Funcion que hace efectivos los cambios de la pregunta en la base de datos
+       post ("/changeQuestion", (request, response) -> {
 
-          openDB();
+         openDB();
 
-          List<Question> cambiar = Question.where("id = "+ (Integer) preguntas.get("ID"));
-          Question pregunta = cambiar.get(0);
+         List<Question> cambiar = Question.where("id = "+ (Integer) preguntas.get("ID"));
+         Question pregunta = cambiar.get(0);
 
-          pregunta.set("pregunta", request.queryParams("cambiar0")).saveIt();
-          pregunta.set("respuestaCorrecta", request.queryParams("cambiar1")).saveIt();
-          pregunta.set("wrong1", request.queryParams("cambiar2")).saveIt();
-          if (request.queryParams("cambiar3") != null) {
-            pregunta.set("wrong2", request.queryParams("cambiar3")).saveIt();
+         pregunta.set("pregunta", request.queryParams("cambiar0")).saveIt();
+         pregunta.set("respuestaCorrecta", request.queryParams("cambiar1")).saveIt();
+         pregunta.set("wrong1", request.queryParams("cambiar2")).saveIt();
+         if (request.queryParams("cambiar3") != null) {
```

Comment:
The value of an input element will never be null, if the input is empty then the value is '' (empty string). So if the value is '' , the correct way should be "pregunta.set("answerX", null).saveIt()" or something like that. 
Otherway in the database, the answer appears as '' and not as null.

Reply:
You wrong.. If the html form is empty, the request.queryParams returns null..

Reply:
You wrong.. If the html form is empty, the request.queryParams returns null..

Reply:
My bad.. I understand it right now